src/main/java/org/folio/rest/impl/OrdersImpl.java

```diff
@@ -117,7 +119,7 @@ public void postOrders(String lang, CompositePurchaseOrder compPO, Map<String, S
 
   @Override
   public void putOrdersById(String id, String lang, CompositePurchaseOrder compPO, Map<String, String> okapiHeaders,
-      Handler<AsyncResult<javax.ws.rs.core.Response>> asyncResultHandler, Context vertxContext) throws Exception {
+      Handler<AsyncResult<javax.ws.rs.core.Response>> asyncResultHandler, Context vertxContext) {
     // TODO Auto-generated method stub
```

Comment:
it annoying to me that sonarqube gives you a red X for TODOs...   In this case we can probably just remove the TODO, since it's just auto-generated, but I'm also fine with keeping it and merging despite SQ